content/patterns/rag-llm-gitops/getting-started.md

```diff
@@ -42,9 +42,14 @@ _Figure 4. Schematic diagram for Ingestion of data for RAG._
 
 _Figure 5. Schematic diagram for RAG demo augmented query._
 
-In Figure 5, we can see RAG augmented query. Llama 2 model is used for language processing, LangChain to
-integrate different tools of the LLM-based application together and to process the PDF
-files and web pages, vector database such as PGVECTOR or REDIS, is used to store vectors, HuggingFace TGI is used to serve the Llama 2 model, Gradio is used for user interface and object storage to store language model and other datasets. Solution components are deployed as microservices in the Red Hat OpenShift cluster.
+In Figure 5, we can see RAG augmented query. Llama 2 model is used for for
```

Comment:
can you remove an extra 'for'

Reply:
Also, we have moved to Mistral-7B instead of Llama 2 due to License restriction, Can you update the 'Llama 2 model' reference to 'Mistral-7B model'

Reply:
I can definitely change it here, but the `rag-llm-gitops` README still reports Llama 2.

Reply:
@gbartolini : Lets update it here. I will create a separate PR to update the README

Reply:
Created the PR https://github.com/validatedpatterns/rag-llm-gitops/pull/33